app/experimenter/experiments/constants/nimbus.py

```diff
@@ -215,6 +215,86 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_USER_URIS = NimbusTargetingConfig(
+    name="Infrequent user (uris)",
+    slug="infrequent_user_uris",
+    description="Between 1 and 6 URIs visited in the past 28 days",
+    targeting="userMonthlyActivity|length >= 1 && userMonthlyActivity|length <= 6 && "
```

Comment:
A quick test of clearing all browsing history does result in `userMonthlyActivity|length` evaluating to 0. Even after browsing a page, at least via asrouter devtools, it evaluates 0 until restarting Firefox. I guess in general, this suggests this data can be stale for 6 hours (frecency update interval). So someone starting firefox on a new day and browsing and closes before 6 hours will likely report 1 fewer day because it likely excludes today's activity?

Reply:
We can look at it another way: when you start firefox on a new day what is your exact classification at that moment (which is also the moment when enrollments are evaluated)? Handling the caching for this targeting will probably not help much because a similar 6 hour refresh window applies to evaluating recipes (`app.normandy.run_interval_seconds`).